guardian/test/utils/junit_xml/element_parser/system_out_parser_test.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:guardian/utils/junit_xml/junit_xml.dart';
+import 'package:test/test.dart';
+import 'package:xml/xml.dart';
+
+import '../test_utils/xml_string_builder_util.dart';
+import '../test_utils/xml_string_parse_util.dart';
+
+void main() {
+  group('SystemOutParser', () {
+    const text = 'Random text output';
```

Comment:
Nit: 
```suggestion
    const text = 'Text output';
```
Just to not confuse readers with "Random"